R/dash.R

```diff
@@ -481,6 +479,13 @@ Dash <- R6::R6Class(
           state=state,
           func=func
         )
+      
+      # # register the callback_context elements
+      # private$callback_context_[[paste(output$id, output$property, sep='.')]] <- list(
+      #   states=setCallbackContext(state),
+      #   triggered=list(),
+      #   inputs=getContextElements(inputs)
+      # )
```

Comment:
what's this commented block?

Reply:
When I initially implemented `callback_context`, I thought it would make sense for the object holding the details to exist before the callback was fired. I've since come to the conclusion that `callback_context` should be `NULL` until the callback is invoked. 

I had not yet removed the lines because it could simplify formatting the context if the object existed already, but I identified a workaround and users report that `callback_context` is fully functional. 

I've removed the inactive statements.

fixed in ba8ecf3